docroot/themes/humsci/humsci_basic/src/scss/ckeditor/imports.scss

```diff
@@ -322,6 +322,11 @@ $px-only: true;
       }
     }
   }
+
+  // Fix Video preview in CKeditor for "Align center and break text" option
+  .drupal-media-style-align-center {
```

Comment:
This always applies to images and I think this is an unintended side effect. Ideally we should add different styles to images and videos. I don't think this is possible with the current markup, but we can add a preprocess to add an extra attribute with the media type. Let's review this next week.

Reply:
Oh yes, you're right! I forgot about images! Yes. I can work on that next week. Thanks!